### DISCUSSION TITLE
What extensions do you use to keep things organized?

### DISCUSSION THREAD

**Discussion by MrUrzza** [vladmandic/sdnext#1228] [General]
As the title says, as your lora, model, etc. count starts to grow, how do you guys keep track of keywords, recommended weights, etc.?

I was using https://github.com/Zetaphor/sd-library-notes but that doesn't appear to play nice with this UI

**Comment by tkiikt** [comment]
I think most people use Civitai. This extension may help. [Civitai Helper](https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper)

**Reply by MrUrzza** [reply]
Thank you, I’ll check it out
